app/web_hub.go

```diff
@@ -539,13 +539,20 @@ func (h *Hub) Start() {
 						}
 					}
 				}
-				if msg.GetBroadcast().UserId != "" {
+
+				if connID := msg.GetBroadcast().ConnectionId; connID != "" {
+					if webConn := connIndex.byConnectionId[connID]; webConn != nil {
+						broadcast(webConn)
```

Comment:
Maybe better to change the function name to `send`? It's somewhat misleading.

Reply:
Happy to change but how does this differ from previous occurrences? Even the channel is called `broadcast` and we are always dealing with a single `WebConn`.

Reply:
Hmm .. fair enough :P